numpy/ma/core.py

```diff
@@ -33,7 +33,7 @@
 import numpy._core.numerictypes as ntypes
 from numpy._core import multiarray as mu
 from numpy import ndarray, amax, amin, iscomplexobj, bool_, _NoValue, angle
-from numpy import array as narray, expand_dims, iinfo, finfo
```

Comment:
this looks like something that could break some downstream code

Reply:
Indeed, it's part of the public API, whether it's a good idea or not. Fixing it.